roadmap.js

```diff
@@ -385,7 +385,7 @@
 
   function _loadFeatureDetectModule() {
     // Be sure to change the preloads in markdown when updating url.
-    const module = import('https://cdn.jsdelivr.net/npm/wasm-feature-detect@1.5/dist/esm/index.js');
```

Comment:
As per the reviews above,

```suggestion
    // Please cache bust by bumping the `v` parameter whenever `feature.json` is
    // updated to depend on a new version of the library. See #353 for discussion.
    // Make sure to also match the preload link in `roadmap.md`.
    const module = import('https://unpkg.com/wasm-feature-detect@1/dist/esm/index.js?v=1');
```